cpp/src/parquet/column_writer-test.cc

```diff
@@ -246,6 +246,18 @@ class TestPrimitiveWriter : public PrimitiveTypedTest<TestType> {
     return metadata_accessor->is_stats_set();
   }
 
+  void metadata_get_stats_has_min_max(bool& has_min, bool& has_max) {
```

Comment:
nit: this should use pointers instead of references for the "out" arguments. I will fix